app/main.py

```diff
@@ -1,13 +1,49 @@
-def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+from dbm import error
```

Comment:
The import statement `from dbm import error` seems unnecessary as `error` is not used anywhere in the code. Consider removing this import to clean up the code.